README.md

```diff
@@ -168,6 +168,60 @@ Por convenção utilizados o seguintes campos dentro do payload de erro:
 - **reason**: mais detalhes do erro
 - **responsecode**: código do erro
 
+## Detalhamento dos atributos
+
+### Payment Request (Pagamento)
+
+`PaymentRequest.java`
+| Atributo                 | Tipo            |  Exemplo                         | Descrição                     | 
+| ------------------------ |:--------------: | :------------------------------: | ----------------------------- |
+| installmentsInterestRate | String          | 0.05                             | Taxa de juros de parcelamento |
+| installments             | String          | 2                                | Número de parcelas | 
+| merchantName             | String          | lojaABC                          | Nome da conta do lojista na VTEX |
+| paymentType              | String          | credit                           | Tipo do pagamento, credito, débito, pix, etc |
```

Comment:
It seems better to list just the possible values: credit and debit, as they are expected instead of translating them